config/local_defaults.go

```diff
@@ -83,7 +83,7 @@ var defaultLocalV4 = Local{
 	SuggestedFeeBlockHistory:              3,
 	SuggestedFeeSlidingWindowSize:         50,
 	TxPoolExponentialIncreaseFactor:       2,
-	TxPoolSize:                            50000,
+	TxPoolSize:                            15000,
```

Comment:
```
// Changing a default value requires creating a new defaultLocalV# instance,
// bump the version number (configVersion), and add appropriate migration and tests.
```

Should we do that before merging this into `master`?

Reply:
I'll fix this tonight is a separate PR. I don't want to delay the testing effort.